lib/rubocop/cli/command/notify_for_extensions.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+module RuboCop
+  class CLI
+    module Command
+      # Run all the selected cops and report the result.
+      # @api private
+      class NotifyForExtensions < Base
+        # Combination of short and long formatter names.
+        INCLUDED_FORMATTERS = %w[p progress fu fuubar pa pacman].freeze
+
+        self.command_name = :notify_for_extensions
+
+        def run
+          return if skip? || extensions.none?
+
+          puts
+          puts 'Notice: Based on detected gems, the following '\
+            'RuboCop extension libraries might be helpful:'
+
+          extensions.each do |extension|
+            puts "  * #{extension} (http://github.com/rubocop-hq/#{extension})"
+          end
+
+          puts
+          puts 'You can opt out of this notification by adding the following to your config:'
```

Comment:
I think notification sounds a bit too formal and we can just go with `message` here.